nav2_bringup/params/nav2_params.yaml

```diff
@@ -148,21 +148,6 @@ map_server:
     use_sim_time: True
     yaml_filename: "turtlebot3_world.yaml"
 
-lifecycle_manager:
-  ros__parameters:
-    use_sim_time: True
-    autostart: True
-    node_names: ['map_server', 'amcl', 'dwb_controller',
```

Comment:
So I think the node names are needed, or are they hard-coded in lifecycle manager? I think if you're moving the others into the launch file (which you did), it would be good to move this one too just to be consistent.

Reply:
I think we should provide the node names to the lifecycle manager through launch, not the params file. I did have them passed to lifecycle manager on `localization_launch` and `navigation_launch` but missed `bringup_launch`. Will push a change shortly.